src/04-modality-specific-files/10-microscopy.md

```diff
@@ -421,14 +421,38 @@ For example:
 {{ MACROS___make_filetree_example(
    {
    "sub-01": {
-      "micr": {
-         "sub-01_sample-01_acq-1_photo.jpg": "",
-         "sub-01_sample-01_acq-2_photo.jpg": "",
+      "ses-01": {
+         "micr": {
+            "sub-01_ses-01_sample-01_acq-1_photo.jpg": "",
+            "sub-01_ses_01_sample-01_acq-2_photo.jpg": "",
+            },
          },
       }
    }
 ) }}
 
+Photo data MAY be accompanied by a JSON file containing the `IntendedFor` field that link
```

Comment:
Should we mention that when there is no JSON or not `IntendedFor` it means that the photo applies to all images ?

Reply:
I could add "In case the photos do not correspond to any particular images, it does not have to be filled." in the description of `IntendedFor` in the table, similar to [fieldmap](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#using-intendedfor-metadata).
I prefer "doesn't apply to a particular image" which is less restrictive than "applies to all images".

> when there is no JSON or not IntendedFor

Just to clarify, the JSON file for photo only contains the `IntendedFor` field.

Reply:
Updated in [124bad7](https://github.com/bids-standard/bids-specification/pull/1000/commits/124bad7ea805ea32bc6a55fe2b2796b6f9045eba)